docs/pages/index.mdx

```diff
@@ -1,29 +1,37 @@
 # Introduction
 
-Mod is a standard for creating Mini-apps. Mod currently only supports the Farcaster protocol, and Mini-apps can be one of two kinds: Mini-apps for creating content, or Mini-apps for displaying embeds.
-Mod Mini-apps are currently defined via a JSON configuration file, and are reviewed and approved into the registry on a case by case basis to ensure security, performance and quality.
+Mod is a standard for creating Mods. Mod currently only supports the Farcaster protocol, and Mods can be one of two kinds: Mods for creating content, or Mods for displaying rich embeds.
```

Comment:
It might make sense to explain "Mods" in terms of miniapps here for the initial definition.

> Mod is a standard for creating mini-apps, i.e. apps that live inside of other apps, which we call Mods.